README.rst

```diff
@@ -103,10 +104,14 @@ Normal Commands Example
 
 You can find more examples in the examples directory.
 
+Note: Make sure you do not reveal your bot token to anyone, they could take control of it.
+
 Links
 ------
 
-- `Documentation <https://pycord.readthedocs.io/en/latest/index.html>`_
-- `Official Discord Server <https://discord.gg/dK2qkEJ37N>`_
-- `Discord Developers <https://discord.gg/discord-developers>`_
-- `Discord API <https://discord.gg/discord-api>`_
+- `Pycord Documentation <https://pycord.readthedocs.io/en/latest/index.html>`_
+- `Pycord Discord Support Server <https://discord.gg/pycord>`_
+- `Discord Developers Support Server <https://discord.gg/discord-developers>`_
+- `Discord API Support Server <https://discord.gg/discord-api>`_
+
+If the Pycord support server invite isn't working, `click here <https://discord.gg/WNS9dKgWza>`_.
```

Comment:
This is unnecessary. "Support server" is too long and vague and previous one seems right. Adding "Pycord" isn't needed either.

Also as Swas said, Don't add a separate line, Just replace the main link with the new one. Make sure it's permanent and has unlimited uses.

Reply:
^this